src/transactions/signed/TransferLockedTokens.ts

```diff
@@ -0,0 +1,43 @@
+import { SignedTransaction } from './SignedTransaction'
+import { LockPeriod } from '../../entities/LockPeriod'
+import { validationRules } from '../../utils/validation'
+
+
+export class TransferLockedTokens extends SignedTransaction {
+  public ENDPOINT: string = 'chain/transfer_locked_tokens'
+  public ACTION: string = 'trnsloctoks'
+  public ACCOUNT: string = 'fio.token'
+  public payeePublicKey: string
+  public canVote: number
+  public periods: LockPeriod[]
+  public amount: number
+  public maxFee: number
+  public technologyProviderId: string
+
+  constructor(payeePublicKey: string, canVote: number, periods: LockPeriod[], amount: number, maxFee: number, technologyProviderId: string = '') {
+    super()
+    this.payeePublicKey = payeePublicKey
+    this.canVote = canVote
+    this.periods = periods
+    this.amount = amount
+    this.maxFee = maxFee
+    this.technologyProviderId = technologyProviderId
+
+    this.validationData = { tpid: technologyProviderId || null }
```

Comment:
could we also add `payeePublicKey` here?
```
this.validationData = { tpid: technologyProviderId || null, payeePublicKey }
```

and in validation.ts
```
  transferLockedTokensRequest: {
    tpid: allRules.tpid,
    payeePublicKey: allRules.fioPublicKey,
  },
```

Reply:
I actually had that, and that validator is failing, will leave it for later if thats ok.

Reply:
hm, ok, let's do not add this, we need to check why it is failing

Reply:
we will complete this work in a follow on story.

Reply:
yes, will do it in a future story